CHANGELOG.md

```diff
@@ -3,7 +3,9 @@
 ## v1.10.0
   {release-date}
 ### Upgrade requirements
-### Migrations
+- [#154](https://github.com/jirik/layman/issues/154) Before upgrade, check names of all users in system and if any of them ended with '_wms', contact the Layman contributors (for example by creating issue at https://github.com/jirik/layman/issues/) and do not upgrade to v1.10.0.
```

Comment:
Do not require manual check. Rather mention that if there appears error message "A workspace has name with reserved suffix" during upgrade, contact contributors. Move it to Migrations and checks.